components/ExampleComponent/ExampleComponent.tsx

```diff
@@ -0,0 +1,10 @@
+const ExampleComponent = () => {
+  return (
+    <div>
+      <h1>Hello from ExampleComponent!</h1>
+      <p>This is a simple example component in Next.js.</p>
+    </div>
+  );
+};
+
+export default ExampleComponent;
```

Comment:
I think named exports are better than default because is it easy to make a mess when you change component name while it is already imported somewhere because for default you can import component with whatever name you want.

Reply:
seems reasonable

Reply:
Done